### DISCUSSION TITLE
Accessibility

### DISCUSSION THREAD

**Discussion by s0kil** [mint-lang/mint#701] [Language Development]
There is little or no discussion about a11y within Mint.

I would like to suggest that we take an approach similar to Svelte providing hints to make the application more accessible.
We could also go further by auto-generating certain attributes for HTML elements.

**Comment by gdotdesign** [comment]
It's something that I wanted to add to Mint but there are bigger issues I need to tackle first. If anybody wants to give this a go I'm open for discussions :slightly_smiling_face:

**Comment by gdotdesign** [comment]
I'm converting this into a discussion.
